### PR TITLE
e2e: hold on to ci-operator logs as artifacts

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: test/e2e/framework/ci-operator.go [hunk 1]
```diff
@@ -131,8 +139,11 @@ func (c *CiOperatorCommand) Run() ([]byte, error) {
 		c.cleanupDone <- struct{}{}
 	}
 	// TODO(skuznets): stream this output?
-	err := c.cmd.Wait()
-	output := b.Bytes()
+	err = c.cmd.Wait()
+	output, readErr := ioutil.ReadAll(tee)
```
Comment (alvaroaleman) [inline test/e2e/framework/ci-operator.go:143]: Why-read it, can't we simply get it from the `log` buffer?

Comment (stevekuznetsov) [reply test/e2e/framework/ci-operator.go:143]: This is a convenient way to write to the file as we read from it

Comment (stevekuznetsov) [reply test/e2e/framework/ci-operator.go:143]: It's not a re-read and is efficient

Comment (alvaroaleman) [reply test/e2e/framework/ci-operator.go:143]: ah sorry, I read the ioutil and though you were reading from the file.

You need to do a `defer logFile.Close()` though

**Comment by stevekuznetsov** [comment]
@alvaroaleman added the defer for closing the log file

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**